gradle/project.gradle.kts

```diff
@@ -75,6 +74,8 @@ allprojects {
 
     repositories {
         jcenter()
+        mavenCentral()
```

Comment:
Why were these repositories added?

Reply:
For ImageIO, somehow jcenter doesn't cover it.